aiidalab_launch/profile.py

```diff
@@ -73,6 +74,14 @@ def __post_init__(self):
     def container_name(self) -> str:
         return f"{CONTAINER_PREFIX}{self.name}"
 
+    def parse_custom_mount(self, custom_mount: str) -> tuple[Path, Path]:
+        # TODO: Add more error handling
+        source, target = custom_mount.split(":")
+        source_path, target_path = Path(source), Path(target)
+        if not source_path.is_dir():
+            raise ValueError("Directory {source} does not exist")
```

Comment:
It could be a volume, no?

Reply:
Good catch, thanks. In `instance.py` we're determining whether we have a bind mount path or a volume name based on `Path.is_absolute()` so this check should work I guess

```python
       if source_path.is_absolute() and not source_path.is_dir():
            raise ValueError(f"Directory {source} does not exist")
```

By the way, I got very confused by this logic initially, and I didn't quite understood the difference between bind mounts and volume mounts. Perhaps the requirement for an absolute path for bind mounts should be mentioned in README / docs....

Reply:
Thanks for the hint. Would you want that additional documentation to target developers or users or both? And were you generally confused about the difference between bind mounts and volumes within docker or just in this context?

Reply:
I somehow thought that volumes are always tied to existing paths (bind), did not know that Docker can manage them internally. It was especially confusing when I first started playing `aiidalab-launch` in default settings, and was very confused that the container was not using my preexisting aiidalab folder in my home folder.

I can try to write something out to README.md for this PR, since I guess we need to document the new functionality anyways.

Reply:
Yes, if you could improve the documentation that would be very much appreciated. Could be as part of this PR.